intern.json

```diff
@@ -1,5 +1,4 @@
 {
-  "tunnel": "null",
```

Comment:
I believe this change broke development environment test running. Was there a corresponding change to `yarn test`?

Reply:
When I've reverted this line and ran it locally, the suite threw an error:

```
Tunnel started
BUG: suiteEnd was received for invalid session 
(ノಠ益ಠ)ノ彡┻━┻
SessionNotCreatedException: [POST http://localhost:4444/wd/hub/session / {"desiredCapabilities":{"name":"intern","idle-timeout":60,"browserName":"firefox","moz:firefoxOptions":{"args":["-headless"]},"browser":"firefox"}}] Unable to create new service: GeckoDriverService
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
```

What needs to be ran to install the `GeckoDriverService` service?

Should we mention [Geckodriver](https://github.com/mozilla/geckodriver) in the README?

Reply:
Ha, this fixed development testing for me, which wasn't working because I didn't have selenium server running (I think). Can you try stopping any selenium / webdriver services you have running?

Reply:
I've stopped all services and re-ran:

```
❯ yarn test
yarn run v1.22.10
$ NODE_OPTIONS=--inspect node src/tests/runner.js
Debugger listening on ws://127.0.0.1:9229/283d0b5c-6f24-4e32-ba84-39d53131fc10
For help, see: https://nodejs.org/en/docs/inspector
Listening on localhost:9000 (ws 9001)
✨  Done in 0.93s.
```

That's suspiciously fast 🤔.

Reply:
try:
```sh
$ rm -rf node_modules && yarn cache clean && yarn install && yarn build
$ yarn test
```